src/ui/settings/data.tsx

```diff
@@ -127,10 +126,9 @@ export const getYouData = () => {
                 type: "route",
                 title: () => s.title,
                 icon: s.icon ? getAssetIDByName(s.icon) : null,
+                usePredicate: s.shouldRender,
```

Comment:
is there a chance we can also fix the developer settings not rendering here? so
`getLayout` would return all screens then `usePredicate: () => useProxy(settings) && s.shouldRender(),`